src/index.html

```diff
@@ -22,6 +22,402 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo-header">
+      <a
+        class="moyo-header__logotype"
+        href="#header"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo-small-logotype"
+        />
+      </a>
+
+      <nav class="moyo-header__navigation">
+        <ul class="moyo-header__list">
+          <li class="moyo-header__item">
+            <a
+              href="#apple"
+              class="moyo-header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="moyo-header__item">
+            <a
+              href="#samsung"
+              class="moyo-header__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="moyo-header__item">
+            <a
+              href="#smartphones"
+              class="moyo-header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="moyo-header__item">
+            <a
+              href="#laptops-computers"
+              class="moyo-header__link"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="moyo-header__item">
+            <a
+              href="#gadgets"
+              class="moyo-header__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="moyo-header__item">
+            <a
+              href="#tablets"
+              class="moyo-header__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="moyo-header__item">
+            <a
+              href="#photo"
+              class="moyo-header__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="moyo-header__item">
+            <a
+              href="#video"
+              class="moyo-header__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Each BEM block should be in a separate file. The header block styles should be in its own file and imported into the main SCSS file.